docs/src/pages/HomePage/home-page.scss

```diff
@@ -131,14 +131,14 @@
 
   &__cards-container {
     display: flex;
-    margin: -$bpk-spacing-xl * 2 auto ($bpk-spacing-xl * 2 - $bpk-card-spacing-desktop);
+    margin: 0 auto ($bpk-spacing-xl * 2 - $bpk-card-spacing-desktop);
```

Comment:
This was the thing making the cards overlap the hero image. We don't need that anymore.